docs/eigenda/operator-guides/eigenda-avs-installation-registration-and-upgrade/holesky.md

```diff
@@ -3,4 +3,188 @@ sidebar_position: 2
 ---
 
 # Holesky
-Coming soon
\ No newline at end of file
+:::info
+Please ensure you have reviewed the [current Active Operator Set cap](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) and ensure you have sufficient delegated restaked ETH TVL before proceeding.
+:::
+
+#### Step 1: Install Prerequisites
+
+- Complete the EigenLayer CLI installation and registration [here](https://docs.eigenlayer.xyz/operator-guides/operator-installation).
+- Install [Docker Engine on Linux](https://docs.docker.com/engine/install/ubuntu/).
+
+#### Step 2: Prepare Local EigenDA files
+
+Clone this repo and execute the following commands:
+
+```
+git clone https://github.com/Layr-Labs/eigenda-operator-setup.git
+cd eigenda-operator-setup/holesky
+cp .env.example .env
+```
+
+Manually update the `.env` file downloaded in the steps above. Modify the sections marked with `TODO` to match your environment.
+
+Create local folders which are required by EigenDA:
+
+```
+mkdir -p $HOME/.eigenlayer/eigenda/holesky/logs
+mkdir -p $HOME/.eigenlayer/eigenda/holesky/db
+```
+
+#### Step 3: Operator Networking Security Setup
+
+Retrieval Setup:
+
+In order for users to retrieve data from your node, you will need to open access to retrieval ports.
+
+Ensure the port specified as `NODE_RETRIEVAL_PORT` in the [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L17) has open access to the public internet.
+
+Dispersal Setup:
+
+In order to limit traffic from the EigenLabs hosted Disperser, please restrict your node's ingress traffic to be allowed by the the list provided below and port number set as `NODE_DISPERSAL_PORT` in the [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L14) in the below setup.
+
+- `3.221.120.68/32`
+- `52.2.226.152/32`
+- `18.214.113.214/32`
+
+#### Step 4: Opt-in into EigenDA
+
+In order to opt-in into EigenDA as an Operator, you must meet the following delegated TVL requirements:
+
+- Have a minimum of 32 ETH delegated.
+- Have more than 1.1x current lowest-stake Operator in the active Operator set. Please see [EigenDA Churn Approver](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) for more detail.
+- The operator to churn out has less than 10.01% of the total stake
+
+Execute the following command to opt-in to EigenDA AVS: 
+This command also downloads the latest SRS points if they don't exist on the node. The file is approximately 8GB in size and the opt-in process can some time to complete depending on the network bandwidth.
+
+```
+./run.sh opt-in
+```
+
+Note: the script will use the `NODE_HOSTNAME` from [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L63) as your current IP.
```

Comment:
update this link to a reference in master branch when https://github.com/Layr-Labs/eigenda-operator-setup/pull/73/files is merged in